openssl/src/dh.rs

```diff
@@ -36,10 +67,15 @@ impl Dh {
         }
     }
 
+
     from_pem!(Dh, ffi::PEM_read_bio_DHparams);
     from_der!(Dh, ffi::d2i_DHparams);
 
+    /// return a DH object for the IETF RFC 5114 value.
```

Comment:
Capitalization

Reply:
The phrasing here could be a bit more clear - maybe something like "Returns the standard 1024 bit MODP group with 160-bit prime order subgroup specified in [RFC 5114](https://tools.ietf.org/html/rfc5114#section-2.1)"

Reply:
Does the following work?
```rust
    /// Return a DH object for the standard 1024 bit MODP group with 160-bit prime order subgroup
    /// specified in [IETF RFC 5114](https://tools.ietf.org/html/rfc5114#section-2.1).
    ///
    /// get_1024_160 corresponds to DH_get_1024_160() in 1.0.2 and 1.1.x
    ///
    /// Requires the `v102` or `v110` features and OpenSSL 1.0.2 or OpenSSL 1.1.0.
    /// [OpenSSL Documentation](https://www.openssl.org/docs/man1.1.0/crypto/DH_get_1024_160.html).
```

Reply:
Yep!